unity/Assets/Scripts/PhysicsSceneManager.cs

```diff
@@ -292,28 +292,55 @@ public class PhysicsSceneManager : MonoBehaviour {
         RequiredObjects.Remove(sop.gameObject);
     }
 
-    public bool SetObjectPoses(ObjectPose[] objectPoses) {
+    public bool SetObjectPoses(ObjectPose[] objectPoses, string errorMessage, bool placeStationary) {
```

Comment:
Is the intent here to update modify the errorMessage parameter so that the caller receives an errorMessage on failure? In which case you need to add the `out` keyword to the parameter.

Reply:
yes, that was the intention. Added the `out` keyword